src/EditorFeatures/Core/Implementation/Suggestions/SuggestedActionWithFlavors.cs

```diff
@@ -50,7 +51,7 @@ public override bool HasActionSets
             // Light bulb will always invoke this property on the UI thread.
             AssertIsForeground();
 
-            if (_actionSets == null)
+            if (_actionSets.IsDefault)
```

Comment:
@Pilchie `ImmutableArray<T>` has an operator declaration for nullable values

https://github.com/dotnet/corefx/blob/master/src/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableArray%601.cs#L91

The compiler warning only comes into play when we promote a normal, non-nullable, operator to a nullable one and one of the values is explicitly null.  Because it's a user defined nullable operator there is no warning here because the implementation could choose to give special meaning to null.